spaceship/spec/connect_api/testflight/testflight_stubbing.rb

```diff
@@ -89,6 +89,11 @@ def stub_build_beta_details
           to_return(status: 200, body: read_fixture_file('build_beta_details.json'), headers: { 'Content-Type' => 'application/json' })
       end
 
+      def stub_build_bundles
+        stub_request(:get, "https://appstoreconnect.apple.com/iris/v1/buildBundles/48a9bb1f-5f0f-4133-8c72-3fb93e92603a/buildBundleFileSizes").
+          to_return(status: 200, body: read_fixture_file('build_bundles_build_bundle_file_sizes.json'), headers: { 'Content-Type' => 'application/json' })
+      end
```

Comment:
Is this right for the tests? The actual URL that I hit when I run locally is https://api.appstoreconnect.apple.com/ but all of these stubs are registered against https://appstoreconnect.apple.com/iris.

I assume this is because we used a private API before the official one? So it should be safe to leave this how it is, but I just wanted to confirm 🙏

Reply:
I actually didn't realize that the tests are using the private API 😱 We should probably able be making tests that hit the official API (at some point).

But this is good for now 😊 Thanks for checking!